duckduckgo/iOS
# Translations for Autofill "Never Save for this Site" feature

Task/Issue URL: https://app.asana.com/0/0/1205159380096691/f
Tech Design URL:
CC:

**Description**:
Translations for "Never save for this Site" autofill functionality



**Steps to test this PR**:
1. Switching your iPhone to another language e.g. French, visit https://fill.dev/form/login-simple and login using credentials you do not have saved
2. In the Save Login prompt confirm the bottom CTA button is localised 
<img width="374" alt="image" src="https://github.com/duckduckgo/iOS/assets/91189922/41456a6a-7dd5-4256-9d8a-b2429d25612d">

3. Go to the `Logins` screen and confirm the `Reset Excluded Sites` row is localised
4. Tap the row and confirm the action sheet presented is localised
<img width="576" alt="image" src="https://github.com/duckduckgo/iOS/assets/91189922/0c2775dc-21ca-4086-b7bf-8d18dfc05133">



**Copy Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
